usaspending_api/awards/models_matviews.py

```diff
@@ -655,7 +655,7 @@ class Meta:
 
 
 class AwardCategory(models.Model):
-    type_code = models.TextField()
+    type_code = models.TextField(db_index=True)
```

Comment:
Index unused in the DB. There are 14 rows in this table. I don't even know where this would be used by the API, I only see old reload scripts using it.